docs/commands/splitmulti.md

```diff
@@ -125,6 +125,6 @@ After this pipeline, the info field "AC" in `1kg.split.vcf.bgz` will have number
 <div class="cmdsubsection">
 ### <a name="splitmulti_annotations"></a> Annotations:
 
- - `va.wasSplit       Boolean` -- this variant was originally multiallelic 
- - `va.aIndex             Int` -- The original index of this variant in imported line, 0 for imported biallelic sites
-</div>
\ No newline at end of file
+ - `va.wasSplit : Boolean` -- this variant was originally multiallelic
+ - `va.aIndex : Int` -- the original allele index (NB: 0 is the reference) of this variant in imported line, 0 for imported biallelic sites
```

Comment:
the original index of this alternate allele in the multiallelic representation (NB: 1 is the first allele, or the aIndex of an originally biallelic variant)... example